app/views/ui/register.html.haml

```diff
@@ -1,37 +1,13 @@
 %section.register.container
   .row
     .col-sm-10.col-sm-offset-1
-      %form.form-horizontal
+      = bootstrap_form_for @user, html: {class: "form-horizontal"} do |f|
```

Comment:
I didn't go back to the UI register form and update the one that was provided I just changed the new register form